packages/vuetify/src/components/VNavigationDrawer/VNavigationDrawer.sass

```diff
@@ -11,67 +11,110 @@
   .v-divider
     border-color: map-get($material, 'dividers')
 
+// Block
 .v-navigation-drawer
-  +bootable()
-
-  display: block
+  -webkit-overflow-scrolling: touch
+  display: flex
+  flex-direction: column
   left: 0
   max-width: 100%
-  overflow-y: auto
-  overflow-x: hidden
+  overflow: hidden
   pointer-events: auto
   top: 0
-  will-change: transform
-  -webkit-overflow-scrolling: touch
   transition-duration: 0.2s
```

Comment:
Variables?

Reply:
There is nothing worth adding to a variable here as the user can either change it locally through props or isn't something I feel would be used very much.